### DISCUSSION TITLE
How to build for older version

### DISCUSSION THREAD

**Discussion by VardenTheOne** [iNavFlight/inav#8494] [Q&A] [answered]
I've followed https://github.com/iNavFlight/inav/blob/master/docs/development/Building%20in%20Docker.md
And I have `inav_6.0.0_HGLRCF722.hex`. But I want to build same one for version 5. How can I do it?

**Comment by DzikuVx** [comment, marked as answer]
`git checkout 5.0.0`

**Reply by VardenTheOne** [reply]
That works, thanks.
But why I can not see that branch in github?

**Reply by DzikuVx** [reply]
because 5.0.0 is a tag not a branch
